modules/ui-extension/src/main/frontend/webpack.config.js

```diff
@@ -28,6 +28,6 @@ module.exports = {
   output: {
     path: __dirname + '/dist/SLING-INF/content/libs/lfs/resources/',
     publicPath: '/',
-    filename: '[name].[contenthash].js',
+    filename: '[name].js',
```

Comment:
We should leave the `[contenthash]`es as is in the codebase.
```suggestion
    filename: '[name].[contenthash].js',
```